### ISSUE TITLE
ngbTooltip on hover throws error: parent.appendChild is not a function

### ISSUE THREAD

**Issue by janvandenberg** [ng-bootstrap/ng-bootstrap#1573]
Since the 1.0.0-alpha.26 I get errors while hovering over ngbTooltip items. This is the stacktrace I get:

core.es5.js:1084 ERROR TypeError: parent.appendChild is not a function
    at EmulatedEncapsulationDomRenderer2.DefaultDomRenderer2.appendChild (platform-browser.es5.js:2789)
    at animations.es5.js:186
    at InjectableAnimationEngine.DomAnimationEngine.onInsert (browser.es5.js:1305)
    at AnimationRenderer.appendChild (animations.es5.js:186)
    at DebugRenderer2.appendChild (core.es5.js:13321)
    at RendererAdapter.createText (core.es5.js:10424)
    at PopupService._getContentRef (popup.js:45)
    at PopupService.open (popup.js:20)
    at NgbTooltip.open (tooltip.js:75)
    at HTMLAnchorElement.<anonymous> (platform-browser.es5.js:2687)
    at ZoneDelegate.webpackJsonp.988.ZoneDelegate.invokeTask (zone.js:398)
    at Object.onInvokeTask (core.es5.js:4140)
    at ZoneDelegate.webpackJsonp.988.ZoneDelegate.invokeTask (zone.js:397)
    at Zone.webpackJsonp.988.Zone.runTask (zone.js:165)
    at HTMLAnchorElement.ZoneTask.invoke (zone.js:460)

I'm using Angular 4.1.3. Tested on Chrome 58.0.3029.110 (64-bit) This is how I'm using the tooltips (pretty straight forward)

<a class="nav-link" (click)="undo()" href="javascript:void(0);" ngbTooltip="Undo change">
   <eva-md-icon>undo</eva-md-icon>
</a>

**Comment by pkozlowski-opensource** [comment]
Please provide a plunker with a reproduce scenario as requested in the issue template. We can't help with issues without a live reproduce scenario and here is why: https://github.com/ng-bootstrap/ng-bootstrap#you-think-youve-found-a-bug

**Comment by pkozlowski-opensource** [comment]
Can't reproduce: http://plnkr.co/edit/hbg0IH3Rs0TrIdmJQThz?p=preview

**Comment by janvandenberg** [comment]
I've tried to create a plunker but the error is not reproduced there: http://embed.plnkr.co/fpyfyLPoh2pc3i2wnTpt/

**Comment by janvandenberg** [comment]
But when I downgrade to 1.0.0-alpha.25 the problem is fixed.

npm install --save @ng-bootstrap/ng-bootstrap@1.0.0-alpha.25

**Comment by pkozlowski-opensource** [comment]
@janvandenberg the only change in the tooltip code for alpha.26 was this: https://github.com/ng-bootstrap/ng-bootstrap/commit/e2c1f906ffa6f880cc761ec41b87e47f805cdd42

I _suspect_ that you _might_ be mixing different Angular versions in your project, but it is impossible to confirm without a reproduce scenario.....

tl;dr; is that we can't take any action here without a plunker reproducing the problem.

**Comment by pkozlowski-opensource** [comment]
Closing for now as we can't do anything without a reproduce scenario. Happy to re-open when we've got a plunker illustrating the issue.

**Comment by tlcoder** [comment]
Same issue here,
